src/state/chromeStore.ts

```diff
@@ -0,0 +1,15 @@
+import { createStore } from 'jotai';
+import { activeModuleAtom, contextSwitcherOpenAtom } from './atoms';
+
+const chromeStore = createStore();
+
+// setup initial chrome store state
+chromeStore.set(contextSwitcherOpenAtom, false);
+chromeStore.set(activeModuleAtom, undefined);
+
+// globally handle subscription to activeModuleAtom
+chromeStore.sub(activeModuleAtom, () => {
+  // console.log('activeModule in store', chromeStore.get(activeModuleAtom));
```

Comment:
I'm assuming we left this in as a POC - but may be cool to be able to log this if we set a debugging flag. Also - this function returns an `unsub` callback - should we have a global handler to invoke all unsub's when the page is being torn down?

Reply:
Yes POC, again, we might not even need the store.